Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/General/Scripts/TargetBehaviors/OnLoadStartScene.cs

```diff
@@ -22,10 +22,7 @@ public void Start()
 
         private void LoadNewScene()
         {
-            if (SceneToBeLoaded != "")
-            {
-                SceneManager.LoadSceneAsync(SceneToBeLoaded, LoadSceneMode.Additive);
-            }
+            MixedRealityToolkit.SceneSystem.LoadContent(SceneToBeLoaded, LoadSceneMode.Additive);
```

Comment:
What if the "SceneToBeLoaded" has not been assigned?

Reply:
The scene system will give you a warning, much like the script used to.